### ISSUE TITLE
use-component example is not working.

### ISSUE THREAD

**Issue by formobi** [Mathpix/mathpix-markdown-it#343]
use-component example not working.
followed these steps
1> git clone https://github.com/Mathpix/mathpix-markdown-it
2> cd mathpix-markdown-it/examples/react-app/use-components
3> npm install, npm start. 
Compiled with problems:
×
ERROR in ../../../lib/contex-menu/menu/menu-item-actions.js 9:11-39
Module not found: Error: Can't resolve 'copy-to-clipboard' in '/Users/imtiyaz/mathpix-markdown-it/lib/contex-menu/menu'
ERROR in ../../../lib/copy-to-clipboard/clipboard-copy-element.js 9:17-67
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/copy-to-clipboard'
ERROR in ../../../lib/helpers/normalize-link.js 9:12-28
Module not found: Error: Can't resolve 'mdurl' in '/Users/imtiyaz/mathpix-markdown-it/lib/helpers'
ERROR in ../../../lib/helpers/parze_link_destination.js 11:18-69
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/helpers'
ERROR in ../../../lib/markdown/dom-adaptor/index.js 9:13-39
Module not found: Error: Can't resolve '@mathpix/domino' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/dom-adaptor'
ERROR in ../../../lib/markdown/highlight/render-rule-highlights.js 11:17-67
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/highlight'
ERROR in ../../../lib/markdown/index.js 114:11-33
Module not found: Error: Can't resolve 'markdown-it' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 127:75-111
Module not found: Error: Can't resolve 'markdown-it-multimd-table' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 130:9-40
Module not found: Error: Can't resolve 'markdown-it-footnote' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 130:46-72
Module not found: Error: Can't resolve 'markdown-it-sub' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 130:78-104
Module not found: Error: Can't resolve 'markdown-it-sup' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 130:110-140
Module not found: Error: Can't resolve 'markdown-it-deflist' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 130:146-173
Module not found: Error: Can't resolve 'markdown-it-mark' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 130:225-251
Module not found: Error: Can't resolve 'markdown-it-ins' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/index.js 133:13-41
Module not found: Error: Can't resolve 'markdown-it-emoji' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/inline-styles/index.js 9:17-67
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/inline-styles'
ERROR in ../../../lib/markdown/md-inline-rule/image.js 9:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-inline-rule'
ERROR in ../../../lib/markdown/md-inline-rule/includegraphics.js 9:27-84
Module not found: Error: Can't resolve 'markdown-it/lib/helpers/parse_link_destination' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-inline-rule'
ERROR in ../../../lib/markdown/md-inline-rule/new-line-to-space.js 9:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-inline-rule'
ERROR in ../../../lib/markdown/md-inline-rule/refs.js 9:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-inline-rule'
ERROR in ../../../lib/markdown/md-inline-rule/underline.js 9:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-inline-rule'
ERROR in ../../../lib/markdown/md-latex-footnotes/block-rule.js 12:12-59
Module not found: Error: Can't resolve 'markdown-it/lib/rules_block/fence.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-latex-footnotes'
ERROR in ../../../lib/markdown/md-latex-footnotes/inline-rule.js 10:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-latex-footnotes'
ERROR in ../../../lib/markdown/md-theorem/inline-rule.js 10:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/md-theorem'
ERROR in ../../../lib/markdown/mdHighlightCodePlugin.js 8:21-44
Module not found: Error: Can't resolve 'highlight.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/mdHighlightCodePlugin.js 9:17-67
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/mdPluginRaw.js 35:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/mdPluginText.js 10:14-61
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/rules.js 18:17-67
Module not found: Error: Can't resolve 'markdown-it/lib/common/utils' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown'
ERROR in ../../../lib/markdown/sanitize/sanitize-html.js 21:20-60
Module not found: Error: Can't resolve 'is-plain-object' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/sanitize'
ERROR in ../../../lib/markdown/sanitize/sanitize-html.js 23:18-41
Module not found: Error: Can't resolve 'parse-srcset' in '/Users/imtiyaz/mathpix-markdown-it/lib/markdown/sanitize'
ERROR in ../../../lib/mathjax/helpers/array/ArrayConfiguration.js 12:25-78
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/Configuration.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/helpers/array'
ERROR in ../../../lib/mathjax/helpers/array/ArrayMappings.js 8:9-58
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/SymbolMap.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/helpers/array'
ERROR in ../../../lib/mathjax/helpers/array/ArrayMappings.js 9:24-76
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/ParseMethods.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/helpers/array'
ERROR in ../../../lib/mathjax/helpers/array/ArrayMappings.js 10:21-70
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/ParseUtil.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/helpers/array'
ERROR in ../../../lib/mathjax/helpers/array/ArrayMethods.js 11:23-79
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/base/BaseMethods.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/helpers/array'
ERROR in ../../../lib/mathjax/helpers/array/ArrayMethods.js 12:21-70
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/ParseUtil.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/helpers/array'
ERROR in ../../../lib/mathjax/index.js 11:32-95
Module not found: Error: Can't resolve 'mathjax-full/js/core/MmlTree/SerializedMmlVisitor.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/index.js 12:19-70
Module not found: Error: Can't resolve 'mathjax-full/js/adaptors/lite/Element.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 10:19-56
Module not found: Error: Can't resolve 'mathjax-full/js/mathjax.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 11:15-54
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 12:16-60
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/Tags.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 13:18-60
Module not found: Error: Can't resolve 'mathjax-full/js/input/mathml.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 14:15-55
Module not found: Error: Can't resolve 'mathjax-full/js/output/svg.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 15:21-66
Module not found: Error: Can't resolve 'mathjax-full/js/input/asciimath.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 16:16-59
Module not found: Error: Can't resolve 'mathjax-full/js/handlers/html.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 17:26-79
Module not found: Error: Can't resolve 'mathjax-full/js/adaptors/browserAdaptor.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 18:23-73
Module not found: Error: Can't resolve 'mathjax-full/js/adaptors/liteAdaptor.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 19:0-51
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/AllPackages.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 22:25-73
Module not found: Error: Can't resolve 'mathjax-full/js/a11y/assistive-mml.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathjax.js 25:29-91
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/base/BaseConfiguration.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/mathml-word.js 10:32-95
Module not found: Error: Can't resolve 'mathjax-full/js/core/MmlTree/SerializedMmlVisitor.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/my-BaseMappings.js 6:9-58
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/SymbolMap.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/my-BaseMappings.js 7:24-76
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/ParseMethods.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/my-BaseMappings.js 8:23-79
Module not found: Error: Can't resolve 'mathjax-full/js/input/tex/base/BaseMethods.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax'
ERROR in ../../../lib/mathjax/serialized-ascii/handlers.js 10:16-63
Module not found: Error: Can't resolve 'mathjax-full/js/core/MmlTree/MmlNode' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/serialized-ascii'
ERROR in ../../../lib/mathjax/serialized-ascii/index.js 10:22-75
Module not found: Error: Can't resolve 'mathjax-full/js/core/MmlTree/MmlVisitor.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/mathjax/serialized-ascii'
ERROR in ../../../lib/sre/sre-browser.js 9:10-50
Module not found: Error: Can't resolve 'speech-rule-engine/lib/sre.js' in '/Users/imtiyaz/mathpix-markdown-it/lib/sre'

**Comment by OlgaRedozubova** [comment]
Fixed in version 2.0.2 (PR #344).
Closing — feel free to reopen if you still experience issues.
